erizo/src/erizo/rtp/LayerDetectorHandler.h

```diff
@@ -16,6 +16,22 @@
 
 namespace erizo {
 
+class LayerInfoChangedEvent : public MediaEvent {
```

Comment:
Maybe we should have all the MediaEvent definitions in the same place? Like `MediaEvents.h`. I see the scenario where we use 15 of these and have to scan through every handler header file to find the definition. It's just an opinion though, feel free to leave it as it is.

Reply:
I'd rather tied them to the classes that make use of them, but we can change it in the future if it's difficult to find them